### DISCUSSION TITLE
Welcome to doodba Discussions!

### DISCUSSION THREAD

**Discussion by yajo** [Tecnativa/doodba#343] [General]
## 👋 Welcome everyone!

We hope you're loving Doodba. Here you can discuss anything related to:

- [Doodba - the image](https://github.com/Tecnativa/doodba).
- [Doodba - the template](https://github.com/Tecnativa/doodba-copier-template).
- [Doodba QA](https://github.com/Tecnativa/doodba-qa).
- [Any other community Doodba project](https://github.com/topics/doodba).
- Stuff related to [Odoo](https://github.com/odoo/odoo) and development or operations around it, that somehow impact on Doodba and the workflow it provides.

We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Enjoy. :blush: 

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ndemarco** [comment]
Breaking the 4-year silence 📢 I'm @NickyDoes. I've been working with ERP systems in manufacturing for ages (think BPCS, DBA Manufacturing, M1). I'm starting my next business 🚀 and am convinced Odoo is a solid foundation. I believe founders should understand their ERP system to unlock its potential. That's why I'm diving deep into Odoo.

Big thanks 🙏 to @yajo, @pedrobaeza, and the Tecnativa team for making this amazing tool available! Anyone else using Odoo to build their business from the ground up?  I'd love to connect and share experiences.

**Reply by pedrobaeza** [reply]
Welcome. You can check on the issues, PRs and discussions activities that there is a decent number of people around using this tool. If you have any specific doubt or issue, just arise it and let people to know it and eventually answer it.
